library/src/test/java/bio/terra/pfb/utils/CompareOutputUtils.java

```diff
@@ -0,0 +1,103 @@
+package bio.terra.pfb.utils;
+
+import static bio.terra.pfb.utils.CompareOutputUtils.FileExtension.JSON;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.pfb.PfbReader;
+import java.io.IOException;
+import java.io.InvalidObjectException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CompareOutputUtils {
+  private static final Logger logger = LoggerFactory.getLogger(CompareOutputUtils.class);
+
+  public static void compareJavaPfbWithPyPfb(
+      String fileName, PfbCommandType commandType, String filePath, FileExtension fileExtension)
+      throws IOException {
+    PfbReader reader = new PfbReader();
```

Comment:
Now that all members of `PfbReader` are `static` you no longer need to create it here, and you can call it below using `PfbReader.showSchema(...)`